daemon/daemon.go

```diff
@@ -145,8 +146,9 @@ func (d *Daemon) chanIdToName(id string) string {
 	if ok {
 		return name
 	}
-	g, err := d.api.GetChannelInfo(id)
+	g, err := d.api.GetConversationInfo(id, false)
```

Comment:
for the record.
https://api.slack.com/methods/channels.info
> This method is deprecated. It will stop functioning in February 2021 and will not work with newly created apps after June 10th, 2020. Learn more.
> Please use these methods instead:
> conversations.info